### DISCUSSION TITLE
Unable to locate the element. Can't call click on element with selector ** because element wasn't found

### DISCUSSION THREAD

**Discussion by pavanshirguppi** [webdriverio/webdriverio#8359] [Technical Discussion]
Why can't webdriverio locate just a few elements down in the viewport? I have reloaded each test case (describe-it test case) to discover the initial elements. scrollIntoView doesn't seem to work either because when it can't find the element, it can't scroll to the element.

**Comment by christian-bromann** [comment]
@pavanshirguppi can you provide a reproducible example?
